### PR TITLE
AdminUI: add Manage Contribution Pages

### PR DESCRIPTION
**Pull request by aydun**
Overview
----------------------------------------
Convert 'Manage Contribution Pages' to SK

Before
----------------------------------------
Old page

After
----------------------------------------
New Page

Note: 
~~- the temporary URL is `civicrm/managecp`~~
- the original page has links to each tab of the edit screen. For consistency, the new page has a single 'Edit' button.

A few things to fix:
1. In the help text, how should we add the help icon and link?
~~2. The New Contribution Page does not work well as a pop-up.  How can the 'Add Contribution Page' button be configured to open normally, not popup?~~
~~3. The Contributions button should have 3 links, but including the token '[CURDATE()]' on the first two prevent them from loading.~~
~~4. The Contribution searches for 'Current Month-To-Date' and 'Fiscal Year-To-Date' rely on this page to calculate the `receive_date_low` and `receive_date_high`.  How can we calculate 'start of this month' and 'start of fiscal year' here?  Or are those better done as presets on the contribution search page?~~
~~5. The Clone button does not work.~~
~~6. The original page has a 'Manage Personal Campaign Pages' button.  Is there a better way than just editing the markup directly?~~
7. It would look better with spacing between the 'Add Contribution Page' and 'Manage Personal Campaign Pages' buttons.

Technical Details
----------------------------------------
Schema xml changes to add paths for add, edit and delete and to specify the `Contribution Page Title` field as html text so it becomes available as filter.

Comments
----------------------------------------

### CODE REVIEW

**Comment by colemanw** [comment]
I tried this out and it basically works, which is great. Some notes:

1. Help text icon, haven't been able to use those yet, but the documentation link should be easy enough to add, just put a normal link in the html.
2. The "create new" and "manage personal campaign pages" are probably better off being done in the afform markup instead of as a property of the search display since it only does poups and it only does one button.
3. Instead of 3 links to the old "Find contributions" screen how about 1 link to a new SearchKit screen?
4. How about putting those search filters on a new SearchKit-based "Find contributions" screen and not trying to be too clever with the links.
5. Yep, the clone button doesn't work. Seems like a problem with the formation of that link?
6. Per `#2` I'd say just edit the markup directly for those 2 buttons.

**Comment by aydun** [comment]
Hey @colemanw  I've eventually got back around to this one!

>     1. Help text icon, haven't been able to use those yet, but the documentation link should be easy enough to add, just put a normal link in the html.

https://github.com/civicrm/civicrm-core/pull/24819 looks like it does some of the work to add in help links.  Can we use that to link to existing text in `.hlp` files?

>     2. The "create new" and "manage personal campaign pages" are probably better off being done in the afform markup instead of as a property of the search display since it only does poups and it only does one button.

Done.  Only thing is no spacing between the buttons.  Is that just a css thing or am I missing something in the html?

>     3. Instead of 3 links to the old "Find contributions" screen how about 1 link to a new SearchKit screen?
For now, I've just linked to the existing 'Find contributions' screen with a param for the `contribution_page_id`

>     4. How about putting those search filters on a new SearchKit-based "Find contributions" screen and not trying to be too clever with the links.
Yes, can break that out as a separate thing.

>     5. Yep, the clone button doesn't work. Seems like a problem with the formation of that link?
Fixed

>     6. Per `#2` I'd say just edit the markup directly for those 2 buttons.
Done

**Comment by aydun** [comment]
Looks like a dodgy test

Test this please

File: ext/civicrm_admin_ui/ang/afsearchManageContributionPages.aff.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "join_entity": null,
+    "title": "Manage Contribution Pages",
+    "description": "",
+    "is_dashlet": false,
+    "is_public": false,
+    "is_token": false,
+    "contact_summary": null,
+    "summary_contact_type": null,
+    "icon": "fa-list-alt",
+    "server_route": "civicrm/managecp",
```
Comment (colemanw) [inline ext/civicrm_admin_ui/ang/afsearchManageContributionPages.aff.json:14]: @aydun time to change the temp URL? I think this PR is mergeable. There are still issues but no blockers IMO.

Comment (aydun) [reply ext/civicrm_admin_ui/ang/afsearchManageContributionPages.aff.json]: Changed & squashed

**Pull request merged**